packages/node_modules/@ciscospark/internal-plugin-board/src/board.js

```diff
@@ -128,6 +128,37 @@ const Board = SparkPlugin.extend({
   },
 
   /**
+   * Deletes a Channel from a Conversation
+   * @memberof Board.BoardService
+   * @param  {Conversation~ConversationObject} conversation
+   * @param  {Board~Channel} channel
+   * @returns {Promise}
+   */
+  deleteChannel(conversation, channel) {
+    return this.spark.request({
+      method: `PUT`,
+      uri: `${channel.aclUrl}/links`,
+      body: this._prepareDeleteChannel(conversation, channel)
+    })
+      .then((res) => res.body);
+  },
+
+  _prepareDeleteChannel(conversation, channel) {
```

Comment:
Is this something likely to be used anywhere other than deleteChannel? if not, not sure it needs to be its own method.